src/components/ImageWithSizeCalculation.js

```diff
@@ -29,6 +30,14 @@ const defaultProps = {
  * it can be appropriately resized.
  */
 class ImageWithSizeCalculation extends PureComponent {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            loading: true,
```

Comment:
@mdneyazahmad let's rename it to `isLoading`
https://github.com/Expensify/App/blob/9ef244cf828f56d31c033df185f1eda4d2c0db28/STYLE.md#boolean-variables-and-props

Reply:
@rushatgabhane done